src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -192,7 +192,7 @@ private void LogSuccessfulTelemetryToClient(AuthenticationResult authenticationR
 
                 telemetryEventDetails.SetProperty(TelemetryConstants.CacheLevel, (int)authenticationResult.AuthenticationResultMetadata.CacheLevel);
               
-                Tuple<string, string> resourceAndScopes  = ParseScopesForTelemetry();
+                (string resource, string scopes) resourceAndScopes  = ParseScopesForTelemetry();
```

Comment:
The tuple values should be used instead of Item1, Item2, etc.

Reply:
sorry. i dont understand